scripts/mainsite.js

```diff
@@ -0,0 +1,213 @@
+var lastStreamer, existingIframe;
+
+window.addEventListener("message", (event) => {
+  if (event.data.eventName == "UPDATE_STATE" && event.data.params.quality && !document.hidden)
+    if (/^((?:160|360|480|720|900|960|1080)p(?:30|48|50|60)|chunked)$/.test(event.data.params.quality))

```

Comment:
nit: Can we create a constant for this regex?

Reply:
And also another variable for `event.data.params.quality`, or deconstruct the usage of them:

```js
const { params } = event.data;
const { quality } = params;
```